angel_system/data/medical/load_bbn_data.py

```diff
@@ -207,42 +231,61 @@ def save_as_kwcoco(classes, data, save_fn="bbn-data.mscoco.json"):
     dset.dump(dset.fpath, newlines=True)
 
 
-def activity_label_fixes(activity_label, target):
-    if activity_label == "put_tourniquet_around":
-        label = "place-tourniquet"
-        label_id = 1
-    if activity_label == "pulls_tight":
-        label = "pull-tight"
-        label_id = 2
-    if activity_label == "secures" and target == "velcro_strap":
-        label = "apply-strap-to-strap-body"
-        label_id = 3
-    if activity_label == "twist" and target == "windlass":
-        label = "turn-windless"
-        label_id = 4
-    if (
-        activity_label == "locks_into_windlass_keeper"
-        or activity_label == "lock_into_windlass_keeper"
-    ):
-        label = "lock-windless"
-        label_id = 5
-    if (
-        activity_label == "wraps_remaining_strap_around"
-        or activity_label == "wrap_remaining_strap_around"
-    ):
-        label = "pull-remaining-strap"
-        label_id = 6
-    if activity_label == "secures" and target == "windlass":
-        label = "secure-strap"
-        label_id = 7
-    if activity_label == "writes_on" and target == "tourniquet_label":
-        label = "mark-time"
-        label_id = 8
-
+def activity_label_fixes(task, activity_label, target):
```

Comment:
This is hacky but I can't think of a better way to map the given activity labels to our version of the label